docs/book/src/topics/managedcluster.md

```diff
@@ -43,8 +43,14 @@ export AZURE_RESOURCE_GROUP="${CLUSTER_NAME}"
 #
 # Otherwise, you can set this value manually.
 #
-export AZURE_SUBSCRIPTION_ID="$(cat ~/sp.json | jq -r .subscriptionId | tr -d '\n')"
+export AZURE_SUBSCRIPTION_ID="$(cat sp.json | jq -r .subscriptionId | tr -d '\n')"
```

Comment:
Can we uncomment [line 38 up above?]( https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2776/files#diff-d3050397e204c2afbdab9b0d5f39824071adb272efb35199bd1350f31eebe1c2R38) 
I overlooked creating `sp.json` because I did not want to run service principle creation twice, also since the above command was commented out.

Reply:
Fair, updated